controllers/installation_controller.go

```diff
@@ -480,14 +480,8 @@ func (r *InstallationReconciler) StartUpgrade(ctx context.Context, in *v1beta1.I
 		return fmt.Errorf("failed to get release bundle: %w", err)
 	}
 	k0surl := fmt.Sprintf(
-		"https://get.k0sproject.io/%[1]s/k0s-%[1]s-amd64", meta.Versions.Kubernetes,
+		"https://replicated.app/embedded-cluster-public-files/k0s-binaries/%s", meta.Versions.Kubernetes,
```

Comment:
I wonder if we shouldn't be reading this URL from [this](https://github.com/replicatedhq/embedded-cluster-operator/blob/main/api/v1beta1/installation_types.go#L50) property. This can be overwritten by the customer in a per license basis. Yes, the property name was poorly named in hindsight.

Reply:
I think we probably should
I'll figure out how to pass that through